python/lsst/ts/linearstage/controllers/zaber_lst.py

```diff
@@ -272,13 +273,12 @@ async def _perform(self, command_name: str, **kwargs: typing.Any):
         """
         if self.device is None:
             raise RuntimeError("Device has not been received.")
-        for axis in self.axes:
-            try:
-                command = getattr(axis, command_name)
-                return await command(**kwargs)
-            except CommandFailedException:
-                self.log.exception(f"{command} rejected for {axis=}.")
-                raise
+        try:
+            command = getattr(axis, command_name)
+            return await command(**kwargs)
+        except CommandFailedException:
+            self.log.exception(f"{command} rejected for {axis=}.")
```

Comment:
If the line of `command = getattr(axis, command_name)` failed, you would not get the `command` and the following line would fail as well:

```
 self.log.exception(f"{command} rejected for {axis=}.")
```

Maybe change to:

```
 self.log.exception(f"{command_name} rejected for {axis=}.")
```